src/js/components/action.jsx

```diff
@@ -57,7 +76,10 @@ export class ActionComponent extends Component {
                                     stripeKey={ publicKeys.stripe }
                                     email={ user.email }
                                     amount={ this.props.amount }
-                                    currency={ this.props.currency.toUpperCase() }>
+                                    currency={ this.props.currency.toUpperCase() }
+                                    name={ stripeAccount.appName }
+                                    image={ stripeAccount.iconUrl }
+                                    closed={ this.onStripeClose.bind(this) }>
```

Comment:
Is there any preference for doing this vs `this.onStripeClose = this.onStripeClose.bind(this)` in the constructor?